C/CUTEst/build_tarballs.jl

```diff
@@ -3,104 +3,42 @@
-export SIFDECODE=${prefix}/SIFDecode
-export CUTEST=${prefix}/CUTEst
+meson compile -C builddir
+meson install -C builddir
 
-# ARCHDefs requires tput
-apk update
-apk add ncurses
-
-# build SIFDecode
-cd $SIFDECODE
-if [[ "${target}" == *-linux* || "${target}" == *-freebsd* || "${target}" == *-mingw* ]]; then
-  echo "6" > sifdecode.opts   # PC64
-  echo "2" >> sifdecode.opts  # Linux
-  echo "6" >> sifdecode.opts  # gfortran
-elif [[ "${target}" == *-apple* ]]; then
-  echo "13" > sifdecode.opts  # macOS
-  echo "2" >> sifdecode.opts  # gfortran
-fi
-echo "nny" >> sifdecode.opts
-./install_sifdecode < sifdecode.opts
-
-# build CUTEst
-cd $CUTEST
-if [[ "${target}" == *-linux* || "${target}" == *-freebsd* || "${target}" == *-mingw* ]]; then
-  echo "6" > cutest.opts   # PC64
-  echo "2" >> cutest.opts  # Linux
-  echo "6" >> cutest.opts  # gfortran
-  echo "2" >> cutest.opts  # build all tools except Matlab
-  echo "8" >> cutest.opts  # gcc
-  export MYARCH=pc64.lnx.gfo
-elif [[ "${target}" == *-apple* ]]; then
-  echo "13" > cutest.opts  # macOS
-  echo "2" >> cutest.opts  # gfortran
-  echo "2" >> cutest.opts  # build all tools except Matlab
-  echo "5" >> cutest.opts  # gcc
-  export MYARCH=mac64.osx.gfo
-fi
-echo "nnydy" >> cutest.opts
-./install_cutest < cutest.opts
-
-# Static libraries
-cp $CUTEST/objects/$MYARCH/single/libcutest.a $prefix/lib/libcutest_single.a
-cp $CUTEST/objects/$MYARCH/double/libcutest.a $prefix/lib/libcutest_double.a
-
-# Shared libraries
-if [[ "${target}" != *mingw* ]]; then
-    extra=""
-    if [[ "${target}" == *-apple-* ]]; then
-        extra="-Wl,-undefined -Wl,dynamic_lookup -headerpad_max_install_names"
-    fi
-    cd $CUTEST/objects/$MYARCH/single
-    gfortran -fPIC -shared ${extra} $(flagon -Wl,--whole-archive) libcutest.a $(flagon -Wl,--no-whole-archive) -o ${libdir}/libcutest_single.${dlext}
-    cd $CUTEST/objects/$MYARCH/double
-    gfortran -fPIC -shared ${extra} $(flagon -Wl,--whole-archive) libcutest.a $(flagon -Wl,--no-whole-archive) -o ${libdir}/libcutest_double.${dlext}
-fi
-
-cp $ARCHDEFS/bin/helper_functions ${bindir}/helper_functions
-cp $SIFDECODE/bin/sifdecoder ${bindir}/sifdecoder
-install_license $CUTEST/lgpl-3.0.txt
+install_license lgpl-3.0.txt
 """
 
-# These are the platforms we will build for by default, unless further
-# platforms are passed in on the command line
-# can't build shared libs on Windows, which imposes all symbols to be defined
-platforms = expand_gfortran_versions(supported_platforms())
-platforms = filter!(p -> !(os(p) == "freebsd" && libgfortran_version(p) == v"3"), platforms)
+# These are the platforms we will build for by default, unless further platforms are passed in on the command line
+platforms = supported_platforms()
+platforms = expand_gfortran_versions(platforms)
 
 # The products that we will ensure are always built
 products = [
-    FileProduct("bin/helper_functions", :helper_functions),
-    FileProduct("bin/sifdecoder", :sifdecoder),
```

Comment:
Why did you remove these file products? It appears that the CUTEst.jl package was providing some functionality using them.